semspaces/space.py

```diff
@@ -10,9 +10,9 @@
 try:
     import pandas as pd
 except ImportError:
-    print 'Warning: pandas not available. Exporting to pandas will not work.'
-
-import space_io
+    print('Warning: pandas not available. Exporting to pandas will not work.')
+# relative import in python 3 - remove references to space_io
+from .space_io import *
```

Comment:
Actually, no. Just a lazy thing I did while writing it and forgot to fix. I've now changed it back to use `space_io`'s namespace. Good call!  
(Sorry for the delay, had to move back and start school up again)